lib/matplotlib/axis.py

```diff
@@ -1826,9 +1826,9 @@ def set_ticklabels(self, ticklabels, *, minor=False, **kwargs):
             ``tick.label2`` if it is visible, in that order.
         """
         ticklabels = []
-            for t in ticklabels:
-                if hasattr(t, 'get_text'):
-                    ticklabels.append(t.get_text())
+        for t in ticklabels:
+            if hasattr(t, 'get_text'):
+                ticklabels.append(t.get_text())
```

Comment:
Wouldn't this have been an indent error? While this is definitely wrong, I don't see how fixing this fixes whatever error you had.

Reply:
Yes, sorry. The indentation was an issue